pkg/pillar/zedcloud/send.go

```diff
@@ -82,6 +82,17 @@ func SendOnAllIntf(ctx ZedCloudContext, url string, reqlen int64, b *bytes.Buffe
 			}
 		}
 		for _, intf := range intfs {
+			portStatus := ctx.DeviceNetworkStatus.GetPortByIfName(intf)
```

Comment:
I guess this is where we need to make the change.. First try sending the message using a reachable interface. If none of them succeed, try the non-reachable interfaces..